posthog/models/web_preaggregated/sql.py

```diff
@@ -0,0 +1,295 @@
+from django.conf import settings
+
+CLICKHOUSE_CLUSTER = settings.CLICKHOUSE_CLUSTER
+CLICKHOUSE_DATABASE = settings.CLICKHOUSE_DATABASE
+
+
+def TABLE_TEMPLATE(table_name, columns, order_by, on_cluster=True):
+    on_cluster_clause = f"ON CLUSTER '{CLICKHOUSE_CLUSTER}'" if on_cluster else ""
+    return f"""
+    CREATE OR REPLACE TABLE {table_name} {on_cluster_clause}
+    (
+        day_bucket DateTime,
+        team_id UInt64,
+        host String,
+        device_type String,
+        {columns}
+    ) ENGINE = AggregatingMergeTree()
```

Comment:
@PostHog/clickhouse Now that we're replacing the data, I think we could default to a `MergeTree` but I am leaving this as it was on the RFC while we have a definition.